src/quo2/components/markdown/text.cljs

```diff
@@ -24,7 +23,8 @@
          {:text-align (or align :auto)}
          (if (:color style)
            style
-           (assoc style :color (if (= (theme/get-theme) :dark) colors/white colors/neutral-100)))))
+           (assoc style :color (or secondary-color
```

Comment:
what is `secondary-color` ? where the name came from, do we have this in figma component ?

Reply:
I wanted to override the default colors we have for the text elements, Like not using `:primary` `:danger` etc .. But rather a parameterize value

Reply:
I wanted to add the color that was in Figma and not from the preset ones
But then designers updated the colors used and I don't think I need this anymore
reverting